apple/ios.bzl

```diff
@@ -251,10 +253,11 @@ def ios_framework(name, **kwargs):
         final framework.
   """
   deps = kwargs.pop("deps", [])
-  apple_dylib_name = "%s.apple_binary" % name
+  bundle_name = kwargs.get("bundle_name", name)
+  apple_dylib_name = "%s.apple_binary" % bundle_name
```

Comment:
This would change the name of the underlying `apple_binary` target, which is risky. It could more easily cause collisions between BUILD targets—especially ones that you can't directly see because they're hidden by macros).

We should already be using `bundle_name` as the binary name [here](https://github.com/bazelbuild/rules_apple/blob/master/apple/bundling/bundler.bzl#L928-L929), so your change should still produce the same result if you revert this one line back to `... % name`. Can you try that and verify?

Reply:
You are right, and it is unnecessary change the apple_binary name.  Reverted that line, Thanks!